dev/io.openliberty.checkpoint_fat_mp/.classpath

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="fat/src"/>
+	<classpathentry kind="src" path="test-applications/JSPWithEL/src"/>
+	<classpathentry kind="src" path="test-applications/appsecurity/src"/>
+	<classpathentry kind="src" path="test-applications/inventory/src/java"/>
+	<classpathentry kind="src" path="test-applications/system/src/java"/>
+	<classpathentry kind="src" path="test-applications/app1/src"/>
+	<classpathentry kind="src" path="test-applications/app2/src"/>
+	<classpathentry kind="src" path="test-applications/ejbapp1/src"/>
+	<classpathentry kind="src" path="test-applications/startupcode/src"/>
+	<classpathentry kind="src" path="test-bundles/test.checkpoint.config.bundle/src"/>
+	<classpathentry kind="src" path="test-applications/timeoutTest/src"/>
+	<classpathentry kind="src" path="test-applications/restClient/src"/>
+	<classpathentry kind="src" path="test-applications/jndiTest/src"/>
+	<classpathentry kind="src" path="test-applications/eventMgr/frontendUI/src/java"/>
+	<classpathentry kind="src" path="test-applications/eventMgr/backendServices/src/java"/>
+	<classpathentry kind="src" path="test-applications/webSocketTest/src"/>
+	<classpathentry kind="src" path="test-applications/openAPItest/src"/>
+	<classpathentry kind="src" path="test-applications/mpjwt/src"/>
+	<classpathentry kind="src" path="test-applications/EJBearApp/src"/>
+	<classpathentry kind="src" path="test-applications/ServletEARapp/src"/>
+	<classpathentry kind="src" path="test-applications/webEARapp/src"/>
+	<classpathentry kind="src" path="test-applications/mphealth/src"/>
+	<classpathentry kind="src" path="test-applications/facesApp/src"/>
+	<classpathentry kind="src" path="test-applications/jsonb/src"/>
+	<classpathentry kind="src" path="test-applications/jsonp/src"/>
+	<classpathentry kind="src" path="test-applications/managedBeans/src"/>
+	<classpathentry kind="src" path="test-applications/mpTelemetry/src"/>
+	<classpathentry kind="src" path="test-bundles/test.checkpoint.bells.bundle/src"/>
+	<classpathentry kind="src" path="test-applications/bells/src"/>
+	<classpathentry kind="src" path="test-applications/virtualhost/src"/>
+	<classpathentry kind="src" path="test-applications/DefaultPrincipalMappingApp/src"/>
+	<classpathentry kind="src" path="test-applications/concurrentApp/src"/>
+	<classpathentry kind="src" path="test-applications/mapCache/src"/>
+	<classpathentry kind="src" path="test-applications/webCacheApp/src"/>
+	<classpathentry kind="src" path="test-applications/xmlBindings/src"/>
+	<classpathentry kind="src" path="test-applications/jaxrspropagation/src"/>
+	<classpathentry kind="src" path="test-applications/mpOpenAPIConfig/src"/>
+	<classpathentry kind="src" path="test-bundles/test-mbeans/src"/>
```

Comment:
Are all these folders really in the new bucket?

Also, shouldn't some folders be removed from the `dev/io.openliberty.checkpoint_fat/.classpath` file for stuff that got moved from there?

Reply:
Thank you for the review Tom! Yes, I have updated the classpath to only include the applications within the new bucket and corrected the .project file for io.openliberty.checkpoint_fat_mp.